### PR TITLE
Cloud and security

### PR DESCRIPTION
**Pull request by mkurek**
Lot of these changes were already accepted. This is merged PR of cloud feature and security info.

### CODE REVIEW

File: docs/installation/configuration.md [hunk 1]
```diff
@@ -121,3 +121,66 @@ To synchronize user list you must run command:
 
 During the process, script will report progress on every 100-th item loaded.
 
+# Migration from Ralph 2
```
Comment (mkurek) [inline docs/installation/configuration.md]: check if this wasn't moved to other file

File: docs/installation/configuration.md [hunk 2]
```diff
@@ -121,3 +121,66 @@ To synchronize user list you must run command:
...
+
+If you used Ralph 2 before and want to save all your data see [Migration from Ralph 2 guide](./data_migration.md#migration_ralph2)
+
+# Synchronization with OpenStack
```
Comment (mkurek) [inline docs/installation/configuration.md]: this should be moved to separated file

File: src/ralph/admin/filters.py [hunk 3]
```diff
@@ -443,14 +445,17 @@ def register_custom_filters():
...
+                isinstance(f, models.ForeignKey) and
+                not getattr(f, '_autocomplete', True)
+            ),
             RelatedFieldListFilter
         ),
         (
             lambda f: isinstance(f, models.ForeignKey),
             RelatedAutocompleteFieldListFilter
         ),
+        (lambda f: isinstance(f, TaggableManager), TagsListFilter),
```
Comment (mkurek) [inline src/ralph/admin/filters.py:458]: @szok why didn't we have this "record" before? :)

File: src/ralph/reports/tests.py [hunk 4]
```diff
@@ -180,9 +180,8 @@ def test_asset_relation(self):
                 'hostname'
             ],
             [
-                str(self.dc_1.id), '', self.dc_1.barcode, self.dc_1.sn,
-                'Keyboard', 'M1', '1', 'None', str(self.dc_1.invoice_date),
-                str(self.dc_1.invoice_no), self.dc_1.hostname
+                self.dc_1.id, None, None, None, 'Keyboard', None, 1, None,
```
Comment (mkurek) [inline src/ralph/reports/tests.py]: revert

**Comment by ar4s** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**